.github/workflows/push-workflow.yaml

```diff
@@ -3,9 +3,12 @@ on: [push]
 name: mabl
 
 jobs:
-  test:
+  test-linux:
```

Comment:
Minor: not sure the job name change makes sense any more.

Nice matrix build simplicity :smile:

Reply:
D'oh. Fixing the name.